_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+- [ITP files from GROMACS](https://userguide.mdanalysis.org/stable/formats/reference/itp)
+
+We have also added [*converters*](https://docs.mdanalysis.org/1.1.1/documentation_pages/converters) to and from other popular analysis packages.
+We plan to expand in this exciting direction in future versions, as laid out in our [interoperability][] roadmap.
+For now, we support:
+- [ParmEd objects](https://userguide.mdanalysis.org/stable/formats/reference/parmed)
+- [the chemfiles library](https://userguide.mdanalysis.org/stable/formats/reference/chemfiles)
+
+New additions to analysis include:
+- `frames` and `times` attributes to AnalysisBase to capture the frames and times that the analysis was `run()` on. This is accessible to all analyses that subclass AnalysisBase.
+- a [correlations module](https://docs.mdanalysis.org/1.1.1/documentation_pages/lib/correlations.html) for computing the discrete autocorrelation function
+- a new [HydrogenBondAnalysis](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/hydrogenbonds.html) class for improved and more efficient analysis of hydrogen bonds, which replaces the now deprecated [hbond_analysis](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/hbond_analysis.html) code
+- an [AverageStructure](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/align.html#MDAnalysis.analysis.align.AverageStructure) class for computing the average structure of a trajectory out of memory
+- a [hole2](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/hole2.html) module for improved interfacing with the HOLE2 program, which replaces the now deprecated [hole](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/hole.html) module
+- a [DensityAnalysis](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/density.html#MDAnalysis.analysis.density.DensityAnalysis) class for improved density analysis, replacing the now deprecated [density_from_Universe()](https://docs.mdanalysis.org/1.1.1/documentation_pages/analysis/density.html#MDAnalysis.analysis.density.density_from_Universe) code
+- a method to compute the root-mean-square-inner-product of subspaces
+- a method to calculate the cumulative overlap of a vector in a subspace
+
+Other additions to core functionality include:
+- [wrap and unwrap on-the-fly transformations](https://docs.mdanalysis.org/1.1.1/documentation_pages/transformations/wrap.html)
+- methods to add [bonds][add_bonds], [angles][add_angles], [dihedrals][add_dihedrals], and [impropers][add_impropers] to a Universe
+- methods to delete [bonds][del_bonds], [angles][del_angles], [dihedrals][del_dihedrals], and [impropers][del_impropers] from a Universe
+- the [get_connections][] method to get bonds, angles, dihedrals and impropers in an AtomGroup, with the ability to specify whether all atoms involved is within the AtomGroup (`outside=False`), or whether *at least one* atom involved is within the AtomGroup (`outside=True`)
```

Comment:
```suggestion
- the [get_connections][] method to get bonds, angles, dihedrals and impropers in an AtomGroup, with the ability to specify whether all atoms involved are within the AtomGroup (`outside=False`), or whether *at least one* atom involved is outside the AtomGroup (`outside=True`)
```

Reply:
(changed within -> outside because that seems like it would be the intention, but I haven't actually looked into `get_connections` so may have misunderstood)

Reply:
Yeah the difference is:

`ag.get_connections("bonds", outside=False)`: both atom a and atom b of the bond are in `ag`

`ag.get_connections("bonds", outside=True)`: at least one of atom a and atom b is *inside* `ag`, but it does not guarantee that at least one atom is *outside*

Reply:
So my take here is the first is -> are is fine, but the second should be `within`?